packages/prepare.js

```diff
@@ -4,38 +4,57 @@ const util = require('util');
 const execFile = util.promisify(require('child_process').execFile);
 
 const rootDir = path.resolve(__dirname, '..');
-const srcDir = path.join(rootDir, 'ed/idl');
-const patchDir = path.join(rootDir, 'ed/idlpatches');
-const dstDir = path.join(rootDir, 'packages/idl');
+
+const packages = [
+  {
+    name: 'idl',
+    srcDir: path.join(rootDir, 'ed/idl'),
+    dstDir: path.join(rootDir, 'packages/idl'),
+    patchDir: path.join(rootDir, 'ed/idlpatches'),
+    fileExt: 'idl'
+  },
+  {
+    name: 'css',
+    srcDir: path.join(rootDir, 'ed/css'),
+    dstDir: path.join(rootDir, 'packages/css'),
+    patchDir: path.join(rootDir, 'ed/csspatches'),
+    fileExt: 'json'
+  },
+];
+
 
 async function main() {
-  // rm dstDir/*.idl
-  const dstFiles = await fs.readdir(dstDir);
-  for (const file of dstFiles) {
-    if (file.endsWith('.idl')) {
-      await fs.unlink(path.join(dstDir, file));
+  for (const { name, srcDir, dstDir, patchDir, fileExt } of packages) {
+    // rm dstDir/*.${fileExt}
+    const dstFiles = await fs.readdir(dstDir);
+    for (const file of dstFiles) {
+      if (file.endsWith('.' + fileExt) && (file !== 'package.json')) {
+        await fs.unlink(path.join(dstDir, file));
+      }
     }
-  }
 
-  // cp srcDir/*.idl dstDir/
-  const srcFiles = await fs.readdir(srcDir);
-  for (const file of srcFiles) {
-    if (file.endsWith('.idl')) {
-      await fs.copyFile(path.join(srcDir, file), path.join(dstDir, file));
+    // cp srcDir/*.${fileExt} dstDir/
+    const srcFiles = await fs.readdir(srcDir);
+    for (const file of srcFiles) {
+      if (file.endsWith('.' + fileExt)) {
+        await fs.copyFile(path.join(srcDir, file), path.join(dstDir, file));
+      }
     }
-  }
 
-  // The patches are against ed/idl/ and can be applied there using `git am`,
-  // but apply them in packages/idl/ instead using `git apply`.
-  // See ed/idlpatches/README.md for how to maintain these patches.
-  const patchFiles = await fs.readdir(patchDir);
-  for (const file of patchFiles) {
-    if (file.endsWith('.patch')) {
-      const patch = path.join(patchDir, file);
-      console.log(`Applying ${file}`);
-      await execFile('git', ['apply', '--directory=packages/idl', '-p3', patch], {
-        cwd: rootDir
-      });
+    // The patches are against srcDir and can be applied there using `git am`,
+    // but apply them in dstDir instead using `git apply`.
+    // See ed/idlpatches/README.md for how to maintain these patches.
+    if (patchDir) {
+      const patchFiles = await fs.readdir(patchDir);
+      for (const file of patchFiles) {
+        if (file.endsWith('.patch')) {
+          const patch = path.join(patchDir, file);
+          console.log(`Applying ${file}`);
```

Comment:
```suggestion
          console.log(`Applying ${patch}`);
```

Otherwise it won't be easy to tell if it's a CSS or IDL patch being applied, their names are the same.